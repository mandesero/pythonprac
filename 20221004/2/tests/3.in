(1,2), (3,4), (2,2), (4,3), (7,0), (1,8), (7, 30), (45, 45), (21, 51), (7, 49), (11, 23)
