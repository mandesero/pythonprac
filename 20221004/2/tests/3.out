((45, 45), (21, 51))
